runelite-api/src/main/java/net/runelite/api/ItemID.java

```diff
@@ -9549,54 +9549,54 @@
 	public static final int STINKHORN_MUSHROOM = 20910;
 	public static final int ENDARKENED_JUICE = 20911;
 	public static final int CICELY = 20912;
-	public static final int ELDER_1 = 20913;
```

Comment:
This file is automatically generated and should not be edited. If you want more readable names then create other constants in the plugin which reference these.